packages/components/src/internal/renderers/DefaultRenderer.tsx

```diff
@@ -50,6 +53,12 @@ export const DefaultRenderer: FC<Props> = memo(({ col, data, noLink }) => {
             // defensively return a MultiValueRenderer, this column likely wasn't declared properly as "multiValue"
             return <MultiValueRenderer data={data} />;
         } else {
+            if (data.has('style') && isConditionalFormattingEnabled()) {
+                style = styleStringToObj(data.get('style'));
```

Comment:
should this use `getDataStyling`?

Reply:
also, should the if statement just check `isConditionalFormattingEnabled()`? it looks like that is what we are doing in the other cases like FileColumnRenderer and UserDetailsRenderer.